ftw/collectionblock/tests/test_hide_more_link_setting.py

```diff
@@ -0,0 +1,41 @@
+from ftw.builder import Builder
+from ftw.builder import create
+from ftw.collectionblock.tests import FunctionalTestCase
+from ftw.testbrowser import browsing
+
+
+class TestHideMoreLinkSetting(FunctionalTestCase):
+
+    def setUp(self):
+        super(TestHideMoreLinkSetting, self).setUp()
+        self.grant('Manager')
+
+        self.page = create(Builder('sl content page').titled(u'A page'))
+
+    @browsing
+    def test_more_link_is_visible_when_option_is_checked(self, browser):
+        create(Builder('sl collectionblock')
+               .titled(u'A collectionblock')
+               .having(show_more_link=True)
+               .within(self.page)
+               .with_default_query())
+
+        browser.login().visit(self.page)
```

Comment:
`login()` is not required, because anonymous users can access the page and its collection block.